pkg/model/manifest_test.go

```diff
@@ -15,18 +15,17 @@ package model
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
-	"testing"
-	"time"
-
 	"github.com/okteto/okteto/pkg/constants"
 	"github.com/okteto/okteto/pkg/discovery"
 	oktetoErrors "github.com/okteto/okteto/pkg/errors"
 	"github.com/okteto/okteto/pkg/model/forward"
 	"github.com/stretchr/testify/assert"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/utils/pointer"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
```

Comment:
```suggestion
	"time"
```
IDE automatically sorted the imports.